www/js/app.js

```diff
@@ -468,9 +468,11 @@ $('#btnAbout').on('click', function () {
     setTimeout(resizeIFrame, 400);
     return false;
 });
-$('input:radio[name=contentInjectionMode]').on('change', function () {
-    // Do the necessary to enable or disable the Service Worker
-    setContentInjectionMode(this.value);
+document.querySelectorAll('input[name="contentInjectionMode"][type="checkbox"]').forEach(function (element) {
```

Comment:
You will need to check that this transpiles correctly for use with IE11, which doesn't support `.forEach` on DOM Elements. Hopefully Babel will handle it, but please check the compiled code by running `npm build-src` and looking in the `bundle.js` code.

Reply:
I checked compiled code but it does contain `.forEach` which is not supported by IE11

Reply:
I have solved this issue [here](https://github.com/RG7279805/kiwix-js/blob/forEach-loop-solution/www/js/app.js#L514) If you think it seems ok I will merge it

Reply:
Although the compiled code contains `.forEach`, core-js probably injected a polyfill that would allow it to work on IE11 in any case, so what's really needed is to test the distribution version on IE Mode in Edge. I don't believe you can use `.forEach` on the util object, because that would cause you to iterate over every property of that object!

The way to solve it without relying on a polyfill is by assigning the selected nodes to a variable, and then using `Array.prototype.slice.call(variable).forEach(...)`. Search in the code of `app.js` for this pattern and copy the pattern.

Reply:
I have tested it on Edge (IE11 mode) and its working fine